### ISSUE TITLE
Not working on IE10

### ISSUE THREAD

**Issue by kailashvele** [a5hik/ng-sortable#283]
Thanks for the  Work its great!
but just noticed that it is not working on IE10 even tried the #262 also the  [Simple Demo](http://a5hik.github.io/ng-sortable/#/) is not working couldn't understand the reason.

**Comment by a5hik** [comment]
@kailashvele are you using the master version. this is having a fix thought it is not released.

**Comment by kailashvele** [comment]
@a5hik  Yes I am using the master version, also while dragging the drag item position is not calculated correctly on IE10 tried containerPositioning: 'relative' but didnt work.

**Comment by kailashvele** [comment]
I guess its the problem of how you structure your code and position attributes to the classes. I have tried to keep it clean and remove the position attribute from styling it worked for me.

**Comment by kailashvele** [comment]
Hi, as updated earlier the structure and position are factors but still it is not working as expected. Also checked the advanced demo on IE10 and here is the screenshot how it looks when dragging on IE10
![untitled](https://cloud.githubusercontent.com/assets/7221983/15531757/a42ddd2c-2279-11e6-99b8-da7b73976bfb.png)
